LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+                                beginning.</note>
+                            <incipit xml:lang="gez">በስመ፡ ሽሐብር፡ ስለከላሂምሸርታ፡ <gap reason="ellipsis"/> በኃይለ፡ ዝንቱ፡ አስማቲከ፡ 
+                                አሸ<supplied reason="undefined" resp="PRS8999Strelcyn"></supplied>ብብሮሙ፡ ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ኵሉ፡ ፍጥረት፡ 
+                                በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i9">
+                            <title ref="LIT7116RingSolomon"/>
+                            <note>Written in the right column under the eighth picture, in the three columns under the ninth picture and in the left 
+                                column under the tenth picture. The text is partly similar to the two prayers above, but using different 
+                                <term key="Magic">magic words (<foreign xml:lang="gez">ʾasmāt</foreign>)</term> at the 
+                                beginning which are said to originate from the ring of <persName ref="PRS8925Solomon"/>.</note>
+                            <incipit xml:lang="gez">በስመ፡ ቀወአረቢ፡ ሂሙፋርክሩክናኤል፡ <gap reason="ellipsis"/> አቅማሐሹን፡ ዘነበረ፡ ውስተ፡ ኅልቀቱ፡ ለሰሎሞን፡ ንጉሥ፡
+                                ከማሁ፡ አሸብሮሙ፡ ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ዛቲ፡ ዓለም፡ በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i10">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect from pagans</title>
+                            <note>Written in the right column under the eighth picture, in the three columns under the ninth picture and in the left 
+                                column under the tenth picture. The text is partly similar to the two prayers above, but using different 
+                                <term key="Magic">magic words (<foreign xml:lang="gez">ʾasmāt</foreign>)</term> at the 
+                                beginning.</note>
+                            <incipit xml:lang="gez">ርብዓም፡ ደርዓም፡ በደርዓም፡ <gap reason="ellipsis"/> በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አሸብሮሙ፡ ለአረሚ፡ በግርማ፡ ገጸ፡ 
+                                ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i11">
+                            <title ref="LIT5888AsmatPrayer">Prayer against enemies</title>
+                            <note>Written in the two middle columns under the tenth picture. The text is another prayer using secret names as those
+                                directly above seeking protection against <persName ref="ETH1508Gallai">Gāllā</persName>, 
+                                <persName ref="ETH2043sanqel">Šānqǝllā</persName> and <foreign xml:lang="gez">ʾArami</foreign> (non-believers).</note>
```

Comment:
Above you translate አረሚ፡ as "pagan" – I think this may be better than "non-believers"